.github/workflows/runner-image-scale-set-build.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and Push Actions Runner scale-set image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "images/ubuntu/dockerfiles-scaleset/**"
+
+  pull_request: # to test the workflow
+    branches:
+      - main
+    paths:
+      - "images/ubuntu/dockerfiles-scaleset/**"
+
+permissions:
+  contents: write
+  packages: write
+  id-token: write
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        base_image: [22.04-devel, 24.04-devel]
+    runs-on: ubuntu-xl
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+          token: ${{ secrets.GITHUB_TOKEN }}
+          persist-credentials: true
+          fetch-depth: 1
+
+      - name: Set up Git
+        shell: bash
+        env:
+          ACCESS_TOKEN: ${{ steps.get_workflow_token.outputs.token }}
+        run: |
+          go env -w GOPRIVATE="github.com/devzero-inc/*"
```

Comment:
small but we prob don't need this part anymore right?
```suggestion
```

Reply:
spot on